proto/process/agent.proto

```diff
@@ -1491,6 +1494,26 @@ message DNSStats {
 message DNSStatsByQueryType {
 	map<int32, DNSStats> dnsStatsByQueryType = 1;
 }
+
+
+message DataStreamsAggregations {
+    message TopicStats {
+        string topic = 1;
+        uint32 count = 2;
+    }
+
+    message KafkaProduceAggregations {
+        repeated TopicStats stats = 1;
+    }
+
+    message KafkaFetchAggregations {
+        repeated TopicStats stats = 1;
+    }
+
```

Comment:
Just for your information, the go type names generated here can be a little awkward. you'll have types called DataStreamsAggregation_TopicStats. You can read the generated code to see how.

Reply:
Ok sounds good thanks!